src/RandomGuess.tsx

```diff
@@ -0,0 +1,28 @@
+interface GuessProps {
+  onKey: (key: string) => void;
+  wordLength: number;
+}
+
+export function RandomGuess(props: GuessProps) {
+  return (
+    <div className="Game-random" aria-hidden="true">
+      <div className="Game-random-guess"
+        tabIndex={-1}
+        role="button"
+        onClick={() => {
+          let guess = "";
+          for (let i = 0; i < props.wordLength; i++) {
+            props.onKey("Backspace")
+          }
+          const chars = "abcdefghijklmnopqrstuvwxyz";
+          for (let i = 0; i < props.wordLength; i++) {
+            let letter = chars.charAt(Math.random() * chars.length)
```

Comment:
Just FYI:

This works because charAt converts the input to an integer, truncating other data by using Math.floor internally.

Relevant spec:

- https://tc39.es/ecma262/multipage/text-processing.html#sec-string.prototype.charat
- https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tointegerorinfinity
- https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tonumber